andela/Ah-backend-guardians
# #162685657  Document API using swagger

**What does this PR do?**
This PR finishes a feature "document API using swagger"

**Description of Task to be completed?**
Document API using swagger

**What are the relevant Pivotal Tracker stories?**
[#162685657](https://www.pivotaltracker.com/n/projects/2232496)

**How should it be manually tested?**
Run each of the following commands in your terminal in the respective order.

1. git clone https://github.com/andela/Ah-backend-guardians.git
2. view the cloned repo in VS code
3. In the terminal of VSCode, git checkout ft-document-api-162685657
4. Activate the virtual environment. (.venv/bin/activate)
5. Pip install -U drf-yasg
6. Run the app python manage.py runserver
7. Hit this URL to access the swagger endpoint. http://127.0.0.1:8000/api/swagger/
8. Login in or register to get the token
9. Click on the authorize button at the top and type "Bearer token"
10. Start testing the other endpoints

You should be able to get back the output as in the screenshot.

Screenshot
![image](https://user-images.githubusercontent.com/41106626/51170555-c980ea80-18bf-11e9-8a35-c55934773d75.png)

After clicking on the Authorize button
<img width="1440" alt="screen shot 2019-01-17 at 12 19 12
 pm" src="https://user-images.githubusercontent.com/41106626/51308071-552a8080-1a52-11e9-833e-991ea9fd4589.png">
